fixbackend/collect/collect_queue.py

```diff
@@ -127,6 +129,7 @@ async def enqueue(
             env=env or {},
             retry_failed_for_seconds=retry_failed_for.total_seconds() if retry_failed_for else None,
         )
+        collect_job.update(kwargs)
```

Comment:
this will put kwargs on root level or?
this is the expected format for azure:
```python
{
        "job_id": "uid",
        "tenant_id": "a",
        "graphdb_server": "b",
        "graphdb_database": "c",
        "graphdb_username": "d",
        "graphdb_password": "e",
        "account": {
            "kind": "azure_subscription_information",
            "azure_subscription_id": "123",
            "tenant_id": "123",
            "client_id": "234",
            "client_secret": "bombproof",
            "collect_microsoft_graph": False,
        },
        "env": {"test": "test"},
    }
```

Reply:
good catch. Updated the PR.